.infrastructure/ingress/ingress.yml

```
@@ -0,0 +1,18 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: todoapp-ingress
+  namespace: todoapp
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /$2
+spec:
+  rules:
+    - http:
+        paths:
+          - path: /blue(.*)
```

What is this path? Where did this `/blue` path come from?
```suggestion
          - path: /
```